### ISSUE TITLE
Missing Predicate in Course Description

### ISSUE THREAD

**Issue by burgess01** [Allegheny-Computer-Science-102-F2020/cs102-F2020-plans#8]
**Describe the bug**

This is an error in the 'Course Description' Section of the cs102-F2020-plans page. The sentence 'An introduction to the foundations of computer science with an emphasis on understanding the abstract structures used to represent discrete objects' is actually a fragment. The fragment is missing its predicate, and so you can fix it by adding a subject: 'This course is an introduction to the foundations of computer science with an emphasis on understanding the abstract structures used to represent discrete objects.'

**State the reproduction steps**

Steps to reproduce the behavior of the bug that you found:

if you go to the cs102-F2020-plans page (https://github.com/Allegheny-Computer-Science-102-F2020/cs102-F2020-plans) and scroll down to the Course Description section, this sentence is the first one in that paragraph.

**Explain the expected behavior**

I expected the text to have correct grammar.

**Explain your execution environment**

I am sitting in my dorm, using a Mac model A2289. I have python version 3.8.5 installed. I am using the VSCode extensions required for the class.

**Comment by gkapfham** [comment]
Hi @burgess01, thanks for raising this issue! Interestingly, the Curriculum Committee at Allegheny College requires us to write the course descriptions in this fashion, perhaps because it avoids the use of a repetitive "This course" in all of the course descriptions. So, if you check all of the course descriptions for all computer science courses, you will see that they all have this mistake!

https://www.cs.allegheny.edu/teaching/bulletin/

In summary, while I agree that this is an issue, I regret that I am not allowed to fix it!
